composer.json

```diff
@@ -15,15 +15,15 @@
         {
             "name": "Branislav Zavracky",
             "email": "branislav.zavracky@brackets.sk",
-            "homepage": "https://www.brackets.sk/en",
+            "homepage": "https://www.brackets.sk",
             "role": "Developer"
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": "^7.2.5"
```

Comment:
@dejwCake is this a breaking change? Do we need to release Craftable v6 becuase of this one?

Reply:
I guess this package is not properly released yet, I am not sure if it is, ask @branislaav. However I think this is not a braking change.